eos/rare-b-decays/lambda-b-to-lambda1520-ll-naive.cc

```diff
@@ -80,7 +80,9 @@ namespace eos
             s_plus = power_of<2>(m_Lb + m_Lstar) - s;
 
         const double alpha_s_mu = model->alpha_s(mu()); // alpha_s at the hard scale
-        const double m_b_msbar = model->m_b_msbar(mu());
+        const double
+            m_b_msbar = model->m_b_msbar(mu()),
+            m_s_msbar = model->m_s_msbar(mu());
```

Comment:
I am puzzled. Wasn't ``m_s_msbar`` needed for the pseudoscalar amplitude? Ah, that only enters in the ``m_l > 0`` case, right?

Reply:
Exact :)